src/Uno.UI/UI/Xaml/Controls/WebView/UnoWKWebView.iOS.cs

```diff
@@ -188,7 +188,9 @@ private void OnRunJavaScriptAlertPanel(WKWebView webview, string message, WKFram
 			var alert = UIKit.UIAlertController.Create(string.Empty, message, UIKit.UIAlertControllerStyle.Alert);
 			alert.AddAction(UIKit.UIAlertAction.Create(OkString, UIKit.UIAlertActionStyle.Default, null));
 
-			controller?.PresentViewController(alert, true, completionHandler);
+			controller?.PresentViewController(alert, true, null);
+
+			completionHandler();
```

Comment:
Why ? :)

Reply:
The completition must be called sooner.  The fix is correct.